inference-engine/tests/functional/inference_engine/ir_serialization/custom_ops.cpp

```diff
@@ -38,7 +38,7 @@ class CustomOpsSerializationTest : public ::testing::Test {
     }
 };
 
-TEST_F(CustomOpsSerializationTest, CustomOpUser_MO) {
+TEST_F(CustomOpsSerializationTest, DISABLED_CustomOpUser_MO) {
```

Comment:
One minor comment which is independent from review. We can use `SKIP_IF_CURRENT_TEST_IS_DISABLED()` taken from `#include <functional_test_utils/skip_tests_config.hpp>` in test body and to add reg-exp to skip tests in [`skip_tests_config.cpp`](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/inference_engine/skip_tests_config.cpp) instead of `DISABLED_`  prefix. All skipped tests in one place is a big advantage

Reply:
Fixed